source/_posts/2017/2017-08-14-how-to-apply-nette-coding-standard-in-your-project.md

```diff
@@ -0,0 +1,109 @@
+---
+id: 50
+layout: post
+title: "How to Apply Nette Coding Standard in Your Project"
+perex: '''
+    Coding standards help you to <strong>focus on architecture issues rather than code details in pull-requests</strong>, moreover in open-source. <strong>I wanted to make this possible in Nette</strong>, but Coding Standards could be found only in <a href="https://nette.org/en/coding-standard">documentation</a>.
+    <br><br>
+    This year I started to work on a Nette Coding Standard (*NCS*) you can put to CLI.
+    <br><br>
+    You'll <strong>be able set it up in in your project either open-source or closed</strong> today.
+'''
+related_posts: [49]
+---
+
+[Nette\CodingStandard](https://github.com/nette/coding-standard/) version [0.5](https://github.com/nette/coding-standard/releases/tag/v0.5.0) with important bug-fixes was released a week ago. This version **is ready to use, includes all important checkers and is used on all `Nette\*` packages in Travis**.
+  
+**NCS checks every pull-request you make to Nette**:
+
+<div>
+    <a href="https://travis-ci.org/nette/application/jobs/261987910#L349">
+        <img src="/assets/images/posts/2017/nette-coding-standard/travis-check.png" class="thumbnail">
+    </a>
+</div>
+
+
+All that need is to [define stage in `travis.yml`](https://github.com/nette/application/blob/2f545e64fc4bfc941d7e48a95e3faca7c468ac35/.travis.yml#L31-L41):
+
+<div>
+    <img src="/assets/images/posts/2017/nette-coding-standard/travis-setup.png" class="thumbnail">
+</div>
+
+That's it! Just 2 commands and it checks any project you have.
+
+But first...
+
+
+## PHP 7.1+
+
+This packages requires PHP 7.1 to run as the rest of the Nette (mostly current `master` or `3.0`).
+
+If you still don't know why should you **join [Symfony, Nette, Doctrine, Zend or Sylius](https://gophp71.org/)**, you can read [this post](/blog/2017/06/05/go-php-71/#why-go-right-to-php-7-1) or wait a bit longer. It's up to you.  
+
+
+## Setup Your Project
+
+You have 2 options how to use NCS in your project. 
+
+
+### 1. As a Composer Project
+
+Nette packages require this approach, because NCS depends on many Nette packages. **NCS should be installed to standalone directory**, so changing the Nette code by NCS doesn't break NCS. Same case as for [PHPStan](https://github.com/phpstan/phpstan).
+
```

Comment:
You're right. It can be installed, just autoload path is not included.

![image](https://user-images.githubusercontent.com/924196/29166523-f2db70c0-7dc6-11e7-843b-a5f958e0b4e3.png)

I'll remove it then.